### DISCUSSION TITLE
Support for Qt6 on Windows other than Win10/11?

### DISCUSSION THREAD

**Discussion by theromorph** [setvisible/DownZemAll#181] [Q&A]
Hello there.

Cann't install DownZemAll on Winsows Server 2012R2. I tried twice before going to Qt site. DownZemAll could not initialize Qt6 libraries becauce Qt6 does not support my OS. Souirce: https://doc.qt.io/qt-6/windows.html
How could I step forward from 2.5.8. to 3.0.2?

**Comment by maxox** [comment]
stick with 2.5.8, or try another downloader

**Comment by setvisible** [comment]
Hello,

2012R2 !!! 

11years ago !!! :-)

Qt6 is probably not compatible with the such old C++ libraries.

Maybe try to install the last MS Visual C/C++ runtime libraries? Check which missing libraries Qt6 is looking for, with Dependency Walker or [Dependencies](https://github.com/lucasg/Dependencies) for instance? But frankly I'm not sure that Windows Server 2012 can run them.

Good luck

Regards

**Comment by theromorph** [comment]
Hello there.
Yep, 2012R2 - I use such things because of luck of newer licenses.

How could I use Dependencies to find missing DLLs? I need some examples.

**Comment by setvisible** [comment]
Run Dependencies and load DownZemAll.exe or the Setup.exe.

=> It will shows errors in case of missing libraries / unfound versions of specific libraries. It indicates sometimes the source of execution crash.

Regards,

**Comment by theromorph** [comment]
Quite confusing. I cann't see any errors. So far, I found some information that qt6 libraries have strict requirements for high windows versions. There are several links to patched versions of qt6core and qt6gui libraries on page https://forum.qt.io/topic/133002/qt-creator-6-0-1-and-qt-6-2-2-running-on-windows-7, but there are no qt6network or qt6widgets pathched versions. And I have no idea, how could I do it by myself. Could you help or do you have any idea about it?

**Comment by setvisible** [comment]
A Windows "Server" is a version of Windows that has generally less functionalities than the classic Windows OS. Microsoft removes a lot of components in the Windows Servers, for security reason (less functionalities limit the exposure), because these versions are intended to run a web server or a specific application.

My guess is that DZA (or one of its dependencies, like Qt6) requires a DLL that isn't installed. Dependency Walker could be helpful to find the missing DLLs.

Or it might be a version issue.

Good luck

**Comment by theromorph** [comment]
There was no luck to launch QT6 under 2012R2. But I've managed to persuade our Windows Administrators Team to upgrade my station to Windows Server 2016. This issue is not actual right now. But it's usefull to notice on your page, that users of older Windows should use 2.5.8 version.

**Comment by regorxxx** [comment]
As already noted, **it should be warned that any QT6 version is essentially broken on windows versions previous to windows 10** (this includes windows 8).

There is no such warning anywhere. Having an update popup when initializing the program telling you to update to a new version without such a warning is a clear way to break some user settings out there.
